index.js

```diff
@@ -276,15 +276,18 @@ ExtractTextPlugin.prototype.apply = function(compiler) {
 									compilation.errors.push(err);
 									return callback();
 								}
-								meta = module[NS];
-								// Error out if content is not an array and is not null
-								if(!Array.isArray(meta.content) && meta.content != null) {
-									err = new Error(module.identifier() + " doesn't export content");
-									compilation.errors.push(err);
-									return callback();
+								// TODO: Investigate "TypeError: Cannot read property '...\node_modules\extract-text-webpack-plugin' of null" in case of css errors and remove workaround check for module.meta[NS] below
```

Comment:
Yes, it don't bugs me, but it seems it's gone for good for everyone. Thus not quite sure how is it better to test this case now, though I'm not an expert in this stuff.
So, closing this PR then.